_data/lifecycle-phases.yml

```
@@ -0,0 +1,159 @@
+phases:
+  - name: Proposal
+    description: |
+      Components under consideration for development through public discussion and a formal proposal.
+    learn_more: Visit the proposal process overview to learn more about how to contribute.
```

Need to add link once it is available.
```suggestion
    learn_more: Visit the [proposal process]() overview to learn more about how to contribute.
```